common/src/PreferenceManager.h

```diff
@@ -197,6 +198,21 @@ namespace TrenchBroom {
         return prefs.get(preference);
     }
 
+    /**
+     * Sets a preference, and saves the change immediately.
+     */
+    template <typename T>
+    void setPref(Preference<T>& preference, const T& value) {
```

Comment:
Ah, finally. I have wanted to add such a function forever, but kept forgetting ;-)